src/components/SelectedFilterOptionList/index.test.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import renderer from "react-test-renderer";
+
+const selectedOptions = ["Ankara", "Istanbul"];
+
+it("SelectedFilterOptionList renders correctly", () => {
```

Comment:
there's no component getting tested here, please render the SelectedFilterOtionList while adding to it the selectedOptions as props to it